backend/src/routes/user.ts

```diff
@@ -148,10 +149,35 @@ router.get("/mentee/:userId", [verifyAuthToken], async (req: Request, res: Respo
       if (!mentee) {
         throw ServiceError.MENTEE_WAS_NOT_FOUND;
       }
+      const {
+        _id,
+        name,
+        imageId,
+        about,
+        grade,
+        topicsOfInterest,
+        careerInterests,
+        mentorshipGoal,
+        pairingId,
+        status
+      } = mentee;
+      let mentorId = "N/A"
+      const pairing = await Pairing.findById(pairingId);
```

Comment:
Currently, our route is making two queries to the mongoDB database (Mentee.findById and Pairing.findById). Look into joining collections in `mongoDB` so that we can merge both these queries into just a single query

https://hevodata.com/learn/mongodb-join-two-collections/

GET mentee and GET mentor are routes which will be frequently used so its important to optimize these.